pysize.py

```diff
@@ -23,8 +23,10 @@ def get_size(obj, seen=None):
         size += sum((get_size(v, seen) for v in obj.values()))
         size += sum((get_size(k, seen) for k in obj.keys()))
     elif hasattr(obj, '__iter__') and not isinstance(obj, (str, bytes, bytearray)):
-        size += sum((get_size(i, seen) for i in obj))
-        
+        try:
+            size += sum((get_size(i, seen) for i in obj))
+        except TypeError:
```

Comment:
I changed the logger level to exception so now the output looks like:

```
ERROR:root:Unable to get size of <class 'collections.OrderedDict'>. This may lead to incorrect sizes. Please report this error.
Traceback (most recent call last):
  File "pysize.py", line 30, in get_size
    size += sum((get_size(i, seen) for i in obj))
TypeError: 'type' object is not iterable
```

Reply:
Great - this also adds the object type which is perfect.